src/test/java/com/microsoft/sqlserver/jdbc/callablestatement/CallableStatementTest.java

```diff
@@ -1294,6 +1309,15 @@ private static void createProcedureManyParams() throws SQLException {
         }
     }
 
+    private static void createProcedureCurrentTime() throws SQLException {
+        String type = manyParamUserDefinedType;
+        String sql = "CREATE PROCEDURE " + currentTimeProc + "@currentTimeStamp datetime = null OUTPUT " +
+                "AS BEGIN SET @currentTimeStamp = CURRENT_TIMESTAMP; END";
```

Comment:
Space needed at start of second string?
```suggestion
        String sql = "CREATE PROCEDURE " + currentTimeProc + " @currentTimeStamp datetime = null OUTPUT " +
                "AS BEGIN SET @currentTimeStamp = CURRENT_TIMESTAMP; END";
```

Reply:
hmm how could this have worked?

Reply:
ok looks like the tests are failing due to this